changelog.txt

```diff
@@ -1,4 +1,7 @@
 V1.XX.X
+[QMS-273] Windows: install Help search index files
+[QMS-270] Windows: rename LICENSE file to 1LICENSE.txt
+[QMS-264] Windows: adapt build scripts for 1.15.1 release
```

Comment:
Please keep order of tickets as in the other sections. Easier to find stuff.e

Reply:
Thanks for the hint. Order should be OK now.